.github/workflows/build-images.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: Build Docker images for all OpenSlides services
+on: push
+env:
+  IMAGE_VERSION: 3.4.0-dev
+jobs:
+  build:
+    name: Builds Docker images
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Build images
+      run: docker/build.sh -D ghcr.io/${{ github.repository_owner }} -t $IMAGE_VERSION all
+
+    - name: Log into registry
+      run: echo "${{ secrets.GITHUB_TOKEN }}" |
+           docker login ghcr.io --username ${{ github.actor }} --password-stdin
+
+    - name: Push images
+      run: |
+        IMAGE_ID=${{ github.repository_owner }}/openslides-server
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
+
+        IMAGE_ID=${{ github.repository_owner }}/openslides-repmgr
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
+
+        IMAGE_ID=${{ github.repository_owner }}/openslides-autoupdate
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
+
+        IMAGE_ID=${{ github.repository_owner }}/openslides-postfix
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
+
+        IMAGE_ID=${{ github.repository_owner }}/openslides-client
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
+
+        IMAGE_ID=${{ github.repository_owner }}/openslides-pgbouncer
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
+
+        IMAGE_ID=${{ github.repository_owner }}/openslides-proxy
```

Comment:
The default does not use https. The readme says, that certs can be included by adding a volume with the certs. I have not test it.

Maybe we can talk about it at a different place. I would like a config, that caddy gets a hostname and automatically generates lets encrypt certificates.

Reply:
Ah, I mean with https://github.com/OpenSlides/OpenSlides/pull/5998 the default changes. In my mind it should already been merged since it fixed a critical issue. I wonder why Gernot didn't merged it yet.

Reply:
Do you think it would be possible to add an option to autogenerate the certs with caddy in #5998 by giving a domain?

Reply:
Not in #5998, but in a new PR it can be done I think.